src/ui/Templates/TemplateCache.ts

```diff
@@ -35,12 +36,28 @@ export class TemplateCache {
     if (publicTemplate && !_.contains(TemplateCache.templateNames, name)) {
       TemplateCache.templateNames.push(name);
     }
+
+    if (pageTemplate && !_.contains(TemplateCache.pageTemplateNames, name)) {
+      TemplateCache.pageTemplateNames.push(name);
+    }
+
     if (defaultTemplate) {
       TemplateCache.defaultTemplates[name] = template;
     }
   }
 
   /**
+   * Register a new page template in the framework. Equivalent to put script tags in the DOM.
+   * @param name
+   * @param template
+   */
+  public static registerPageTemplate(name: string, template: Template);
```

Comment:
Hum this method does nothing ?

It simply calls `registerTemplate` directly, and does pretty much no treatment.

Why can't you simply call `registerTemplate` with the correct parameters ?